### ISSUE TITLE
Update Screenshot.png

### ISSUE THREAD

**Issue by jordesign** [WordPress/twentyseventeen#119]
I was thinking adding the screenshot would be a _perfect_ first pull request as a new contributor - but then was double checking the guidelines.

'The screenshot should be of the actual theme as it appears with default options'
Is this meant to be with the Theme Unit Test content?

My current appearance using WPTest.io content is this
![image](https://cloud.githubusercontent.com/assets/1842363/18975890/f43adbf4-86f0-11e6-8992-10ea08180b87.png)

**Comment by josephfusco** [comment]
It doesn't need theme unit test content, but we should finish implementing the design first and have proper dummy content in place. :) Good to leave this open though.

**Comment by jordesign** [comment]
Fair Call - 
So this can wait until #4 is finalised and closed

**Comment by josephfusco** [comment]
@melchoyce should we update the default custom header image to the one in the design for this? I can make a separate issue for it.

Currently using:

![header](https://cloud.githubusercontent.com/assets/6676674/19335389/da268fda-90d0-11e6-8289-7ac000e8ba5e.jpg)

**Comment by jordesign** [comment]
Seems it would be worthwhile - have we finalised the default custom content.

**Comment by melchoyce** [comment]
Waiting on #134.

**Comment by celloexpressions** [comment]
Once we have the header image in, we need to decide what the site title and tagline should be, whether there should be a logo, and what menu items should be visible. I think we'll be able to put that content in and take a screenshot directly at 1200x900px and should be good to go. 

We can collaborate on the content shown in the screenshot here. I'd like to see a logo, since this is geared at businesses. I think a smallish white-on-transparent WordPress logo (no text) would make sense.

**Comment by celloexpressions** [comment]
Here's a first pass (at the required 1200x900 size):
![screenshot](https://cloud.githubusercontent.com/assets/7875961/19412185/0a35040e-92c5-11e6-87f1-4fdfc5c50782.png)

We need to decide how big logos should be in #345. I used "Twenty Seventeen" for the site title, and "Bringing your site to life" as the tagline (pulled from the theme description). However, this highlights the difficulty of reading the tagline. I still think it needs either to match the title color without an opacity reduction, or to be made larger.

I used a menu with similar items to what I used in a screenshot for one of my business-oriented themes for now, but I think that's the main decision to make. We should also consider if the logo should be a different color, although I think it's the right logo since the roundness plays off the vases in the image nicely. I tried it in WordPress blue and it looked considerably worse. For the purposes of the screenshot, this logo is framed nicely if it's right at 200px square:
![screenshot](https://cloud.githubusercontent.com/assets/7875961/19412197/63cc8ed8-92c5-11e6-894e-2794873b0f34.png)

**Comment by enodekciw** [comment]
Personally, I'd love to see more theme in a screenshot, not just the menu.

**Comment by melchoyce** [comment]
I'd really like to address https://github.com/WordPress/twentyseventeen/issues/189#issuecomment-252521199 before taking a screenshot. I also think the black text and tagline is super hard to read. 

I was thinking something more like this:

![screen shot 2016-10-15 at 2 28 15 pm](https://cloud.githubusercontent.com/assets/2846578/19412421/e48d433c-92e3-11e6-96f8-4248e79163dd.png)

(With better dummy content)

**Comment by celloexpressions** [comment]
Always-floating the logo works for me, although I did kind of like the idea of putting it above when there's so much space on the front page. (#189 is specifically about pages other than the front page, but we could change it here also).

If the text is that low on the image (and over the gradient), white works, but further up black provides much higher contrast. We could play with the gradient to improve that maybe.

**Comment by laurelfulford** [comment]
Screenshot added in #463.
